packages/salesforcedx-utils-vscode/src/telemetry/telemetryReporter.ts

```diff
@@ -8,12 +8,12 @@ import * as appInsights from 'applicationinsights';
 import * as fs from 'fs';
 import * as os from 'os';
 import * as path from 'path';
-import * as vscode from 'vscode';
```

Comment:
so this doesn't work for webpack? `import * as vscode from 'vscode'`

Reply:
This change wasn't specifically for webpack but for finding which things need to be mocked when writing the tests.